base.requirements.txt

```diff
@@ -1,4 +1,5 @@
 ## Base requirements
+dojson==0.4.0
```

Comment:
It is picked up when `-e .` is run as part of `requirements.txt`, so I agree with @jmartinm  that it should be pinned in setup.py.